examples/with_winit/src/multi_touch.rs

```diff
@@ -1,3 +1,6 @@
+// Copyright 2023 the Vello Authors
```

Comment:
I'm not sure this is right. It's mostly the work of the egui authors

Reply:
Good catch! I've created #464 to address this.